yarn-project/circuit-types/src/stats/stats.ts

```diff
@@ -189,7 +189,9 @@ export type NoteProcessorStats = {
   /** How many notes have been seen and trial-decrypted. */
   seen: number;
   /** How many notes had decryption deferred due to a missing contract */
-  deferred: number;
+  deferredIncoming: number;
+  /** How many notes had decryption deferred due to a missing contract */
+  deferredOutgoing: number;
```

Comment:
Had to add deferred outgoing since because we now call the  'compute_note_hash_and_optionally_a_nullifier` function for outgoing as well.